include/lp_oracles/solve_lp.h

```diff
@@ -34,7 +34,7 @@
 
 // compute the chebychev ball of an H-polytope described by a dxd matrix A and  d-dimensional vector b, s.t.: Ax<=b
 template <typename NT, typename Point, typename MT, typename VT>
-std::pair<Point,NT> ComputeChebychevBall(MT const& A, VT const& b){
+std::pair<Point,NT> ComputeChebychevBall(const MT &A, const VT &b){
```

Comment:
I see that there are many styles in the library, I remember many places using `MT const& A` to emphasise on type